src/pages/App.tsx

```diff
@@ -59,6 +60,14 @@ function Main() {
               setActiveAccount={setActiveAccount}
             />
           </Route>
+          <Route path="/human/bids">
+            <Bids
+              accounts={accounts}
+              activeAccount={activeAccount}
+              setAccounts={setAccounts}
+              setActiveAccount={setActiveAccount}
+            />
+          </Route>
```

Comment:
After merging https://github.com/KappaSigmaMu/ksm-app/pull/30 this Route will need to be updated